### PR TITLE
Refactor module imports/exports

### PR DESCRIPTION
**Pull request by brunoBrizo**
#### Description:

* Refactor modules to only import other Modules, not repos etc.
* Add Service methods to be reused by other modules

---

#### Notes:
   -

### CODE REVIEW

**Review by pablanco — APPROVED** [review]
Nice refactor Bruno!

File: src/services/order/order.service.ts [hunk 1]
```diff
@@ -58,7 +56,7 @@ export class OrderService {
     await Promise.all([
```
Comment (CamilaNiebles) [inline src/services/order/order.service.ts:56]: The await is not necessary inside the array. PromiseAll is useful when you need to send multiple transactions and you don't need a sequence.

**Review by CamilaNiebles — COMMENTED** [review]
Hey Bruno! good job here! One more question, is it really necessary to use a folder for each module?
Maybe you can think of something like 
```
modules/
     - cart.module.ts
     - order.module.ts
     - etc.
 ```

File: src/services/order/order.service.ts (hunk 1 above)
Comment (brunoBrizo) [reply src/services/order/order.service.ts:56]: Yeah that was a copy/paste mistake. Thanks for catching that!

**Pull request merged**